extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/handler/AstroThingHandler.java

```diff
@@ -287,40 +317,49 @@ private boolean isPositionalChannelLinked() {
     public void triggerEvent(String channelId, String event) {
         if (getThing().getChannel(channelId) != null) {
             triggerChannel(getThing().getChannel(channelId).getUID(), event);
-        } else {
-            logger.warn("Event {} in thing {} does not exist, please recreate the thing", event, getThing().getUID());
+            return;
```

Comment:
But my personal opinion in having if..else combination makes it more cumbersome. You are right. It should have been better to have it in a different PR but I tried to have such small changes together to reduce the time to check PRs.